catalogs/catalog-jdbc-doris/src/main/java/com/datastrato/gravitino/catalog/doris/converter/DorisExceptionConverter.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.catalog.doris.converter;
+
+import com.datastrato.gravitino.catalog.jdbc.converter.JdbcExceptionConverter;
+import com.datastrato.gravitino.exceptions.GravitinoRuntimeException;
+import java.sql.SQLException;
+
+/** Exception converter to gravitino exception for Doris. */
```

Comment:
`gravitino` -> `Gravitino `

Reply:
same typo in `MysqlExceptionConverter` , i can fix it in another PR (for #2553 )